Assets/MRTK/SDK/Features/UX/Scripts/InteractiveElement/BaseInteractiveElement.cs

```diff
@@ -306,14 +451,38 @@ public void AddNearInteractionTouchable()
         }
 
         /// <summary>
-        /// Checks if a state is currently in the State list. This method is specifically used for checking the 
-        /// contents of the States list during edit mode as the State Manager contains runtime methods. 
+        /// Register the IMixedRealityPointerHandler for global input when the SelectFar state is
+        /// present on Start and the Global property is true.
```

Comment:
Again, it's unexpected that global needs to be set here, it's not clear from the method name, but also feels a little unexpected to me.